includes/admin/settings/class-settings-bootstrap.php

```diff
@@ -1130,6 +1130,16 @@ private static function settings_appearance() {
     }
 
     private static function settings_payment() {
+		wpbdp_register_setting(
+			array(
+				'id'      => 'payments-on',
+				'type'    => 'hidden',
+				'class'   => 'hidden',
+				'default' => false,
```

Comment:
Let's switch the default on this to true. I think that would be a better fallback in case add-ons are using this check.